examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -18,60 +18,13 @@
  */
 
 const app = {
-  isInitialized: false,
   // Application Constructor
   initialize: function() {
     document.addEventListener(
       "deviceready",
       this.onDeviceReady.bind(this),
       false
     );
-    // this method gets called after configure is done
-    // or when the customerInfo is updated from a purchase, restore, login / out or renewal
-    window.addEventListener("onCustomerInfoUpdated", (info) => {
-      if (this.isInitialized) { return; }
-      this.isInitialized = true;
```

Comment:
the previous solution relied on this event, but the callback simplifies this part